src/app/app.component.html

```diff
@@ -1,3 +1,3 @@
-<div>
-  {{ allTeams | async }}
+<div *ngFor="let team of allTeams$ | async">
```

Comment:
Good use of the ngFor structural directive! We should look into trackBy functions for these, I haven't used them yet personally (yet!), but apparently it makes the directive more performant and is good practice.